cmd/redirector/main.go

```diff
@@ -224,7 +224,7 @@ func main() {
 	if _, err := maxprocs.Set(maxprocs.Logger(func(format string, v ...interface{}) {
 		logger.Info(fmt.Sprintf(format, v...))
 	})); err != nil {
-		logger.Warn("Setting GOMAXPROCS failed: %s", err)
+		logger.Warn(fmt.Sprintf("Setting GOMAXPROCS failed: %s", err))
```

Comment:
[imo]
It appears that this issue was detected by linting due to updates in Go v1.22.
https://tech.andpad.co.jp/entry/2024/03/22/100000

While it is possible to avoid linting failures using format strings as shown in this PR, it could result in the loss of some information in structured logs.

By passing key-value pairs as shown below, you can resolve the issue pointed out by the linter. This approach is more desirable in my opinion.

```bash
logger.Warn("Setting GOMAXPROCS failed", "err", err)
```

Thank you for the PR!

Reply:
https://go.dev/play/p/jfv-z1U3PQT

Reply:
thank you. I fixed it with the following commit.

https://github.com/livesense-inc/go-simple-http-redirector/pull/17/commits/6266697d047e48f420f3af9ce735d4306fb93c13